src/main/java/com/uid2/admin/vertx/service/SiteService.java

```diff
@@ -112,7 +112,7 @@ private void handleSiteList(RoutingContext rc) {
                 jo.put("id", site.getId());
                 jo.put("name", site.getName());
                 jo.put("enabled", site.isEnabled());
-                jo.put("types", site.getClientTypes());
+                jo.put("Client Types", site.getClientTypes());
```

Comment:
is this capitalization intentional and identical to the other place? where is that other place btw -  should we be deduplicating?